bittide-tools/hitl/config-gen/Main.hs

```diff
@@ -48,15 +49,16 @@ data Config = Config
 configs :: IO [Config]
 configs = sequence
   [ -- Generate config based on Haskell definitions
-    makeConfig 'FincFdec.fincFdecTests                 FincFdec.tests
-  , makeConfig 'FullMeshHwCc.fullMeshHwCcTest          FullMeshHwCc.tests
-  , makeConfig 'FullMeshHwCc.fullMeshHwCcWithRiscvTest FullMeshHwCc.tests
-  , makeConfig 'FullMeshSwCc.fullMeshSwCcTest          FullMeshSwCc.tests
-  , makeConfig 'SyncInSyncOut.syncInSyncOut            SyncInSyncOut.tests
-  , makeConfig 'Transceivers.transceiversUpTest        Transceivers.tests
-  , makeConfig 'VexRiscv.vexRiscvTest                  VexRiscv.tests
-  , makeConfig 'BoardTest.boardTestSimple              BoardTest.testsSimple
-  , makeConfig 'BoardTest.boardTestExtended            BoardTest.testsExtended
+    makeConfig 'FincFdec.fincFdecTests                  FincFdec.tests
+  , makeConfig 'FullMeshHwCc.fullMeshHwCcTest           FullMeshHwCc.tests
+  , makeConfig 'LinkConfiguration.linkConfigurationTest LinkConfiguration.tests
```

Comment:
I know these weren't ordered before, but since we're diffing up this whole block anyway: could you order them?